lib/file.js

```diff
@@ -134,6 +134,10 @@ class File extends EventEmitter {
     render.render(this, elem, opts, cb)
   }
 
+  includes (piece) {
```

Comment:
This looks useful! Can you please add this method in the docs?

Reply:
will do